storage/txcache/txCacheFailsafe.go

```diff
@@ -0,0 +1,140 @@
+package txcache
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+var _ storage.Cacher = (*txCacheFailsafeDecorator)(nil)
+var _ txCache = (*txCacheFailsafeDecorator)(nil)
+
+// txCacheFailsafeDecorator is a decorator over a cache, a decorator which wraps some operations in try / catch constructs
+type txCacheFailsafeDecorator struct {
```

Comment:
Why do we need this fail safe cache? Why the wrapped backing cache is not safe?

Reply:
Removed, see above.